.github/workflows/test-scaffold.yml

```diff
@@ -0,0 +1,161 @@
+        DREVOPS_CI_TEST_RESULTS: ${{ env.DREVOPS_CI_TEST_RESULTS }}
+        # Directory to store test artifacts.
+        DREVOPS_CI_ARTIFACTS: ${{ env.DREVOPS_CI_ARTIFACTS }}
+        # Check only minimal stack requirements.
+        DREVOPS_DOCTOR_CHECK_MINIMAL: 1
+        # Directory to store code exported between jobs.
+        DREVOPS_EXPORT_CODE_DIR: /tmp/workspace/code
+
+    steps:
+      -
+        name: Checkout the repo
+        uses: actions/checkout@main
+
+      -
+        name: Check Docker and Compose version
+        run: |
+          echo "[INFO] Docker version."
+          docker version
+          echo "[INFO] Docker Compose version."
+          docker compose version
+
+      -
+        name: Setup Docker network
+        run: docker network prune -f >/dev/null 2>&1 && docker network inspect amazeeio-network >/dev/null 2>&1 || docker network create amazeeio-network >/dev/null 2>&1 || true
+
+      -
+        name: Process codebase to run in CI
+        run: |
+          find . -name "docker-compose.yml" -print0 | xargs -0 -I {} sh -c "sed -i -e ''/###/d'' {} && sed -i -e ''s/##//'' {} && sed -i -e 's#- .:/app:delegated#[]#' {}"
+          mkdir -p "${DREVOPS_EXPORT_CODE_DIR}"
+
+      -
+        name: Init scaffold
+        run: ahoy init scaffold-test ${{ matrix.project_type }} ${{ env.GOVCMS_VERSION }}
+
+      -
+        name: Build project
+        run: ahoy up
+        env:
+          GOVCMS_IMAGE_VERSION: ${{ matrix.govcms_image }}
+          GOVCMS_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      -
+        name: Install GovCMS profile
+        run: ahoy install
+
+  notify:
+    runs-on: ubuntu-latest
+    needs: [ test ]
+    steps:
+      -
+        name: Notify failures to Slack
+        id: failure_slack
+        uses: slackapi/slack-github-action@v1.27.0
+        with:
+          # See https://github.com/slackapi/slack-github-action?tab=readme-ov-file
+          channel-id: ${{ secrets.SLACK_CHANNEL }}
+          payload: |
+            {
+              "attachments": [
+                {
+                  "color": "#ff0000",
+                  "blocks": [
+                    {
+                      "type": "header",
+                      "text": {
+                        "type": "plain_text",
+                        "text": "Scaffold test failure"
+                      }
+                    },
+                    {
+                      "type": "context",
+                      "elements": [
+                        {
+                          "type": "mrkdwn",
+                          "text": "Some jobs were not successful, please view the pipeline and rerun."
+                        }
+                      ]
+                    },
+                    {
+                      "type": "section",
+                      "fields": [
+                        {
+                          "type": "mrkdwn",
+                          "text": "*Project Type:*\n${{ matrix.project_type }}"
```

Comment:
@steveworley the matrix is not available here. The `notify` is a separate job so that GHA only sends one Slack message upon failure. If we use notify as a step of the previous job, GHA will send 4 Slack messages if all 4 jobs fail.